docs/guides/_common/ory-cli.mdx

```diff
@@ -1,18 +1,21 @@
-The Ory CLI includes useful functionality to manage your Ory Cloud Project. But
-that is not why we require it in this guide!
+Ory CLI provides a convenient way to configure and manage Ory Cloud projects. Additionally, the CLI 
+contains Ory Proxy - a reverse proxy that rewrites cookies to match the domain your application is currently on. 
 
-Ory's philosophy is to make hard things easy for you. For this reason, Ory has
-deployed measures against all
-[OWASP Top 10](https://owasp.org/www-project-top-ten/) and implements the
-[OWASP Authentication Cheat Sheet](https://cheatsheetseries.owasp.org/cheatsheets/Authentication_Cheat_Sheet.html)
-along other mechanisms.
+Ory Proxy is a reverse proxy deployed in front of your application. The Proxy mirrors Ory endpoints on the same 
+domain as the application you're running and rewrites cookies to match the domain your application is currently on.
 
-Therefore, Ory manages Anti-CSRF Cookies as well as Ory Session Cookies for you.
-That however requires that Ory and your application run on the same domain!
+As a result, the origin of the cookies is set correctly to the domain you run the app on instead of <br /> 
+`<your-project-slug>.projects.oryapis.com.`
```

Comment:
You can add newlines using two white spaces at the end:

```
instead of  
```

however, it usually does not look very great visually to have a singular code box, so it is usually better to have a code box for this (as it allows you to copy the code also):

````suggestion
As a result, the origin of the cookies is set correctly to the domain you run the app on instead of <br /> 

```
<your-project-slug>.projects.oryapis.com.
```
````

Reply:
Oh, I did that bc for some reason docusaurus broke the address into two lines which made it look weird. 
Makes sense to put it into a code block, I'll commit your suggestion.

Reply:
👍